src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlCommandExtended.cs

```diff
@@ -0,0 +1,203 @@
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using static Interop.VoiceControlManager;
+using static Interop.ControlCommandExpand;
+
+namespace Tizen.Uix.VoiceControlManager
+{
+    /// <summary>
+    /// This Class represents the Extended Command
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class VoiceControlCommandExtended : VoiceCommand
+    {
+        private VoiceCommand _cmd;
+
+        /// <summary>
+        /// Public Constructor
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <param name="cmd">Voice Command</param>
+        public VoiceControlCommandExtended(VoiceCommand cmd)
+        {
+            _cmd = cmd;
+        }
+
+        /// <summary>
+        /// Gets or Sets command domain.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <exception cref="UnauthorizedAccessException">This exception can be due to permission denied.</exception>
+        /// <exception cref="NotSupportedException">This exception can be due to not supported.</exception>
+        public int Domain
+        {
+            get
+            {
+                int domain;
+
+                ErrorCode error = VcCmdGetDomain(_cmd._handle, out domain);
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "GetDomain Failed with error " + error);
+                    return 0;
+                }
+
+                return domain;
+            }
+            set
+            {
+                ErrorCode error = VcCmdSetDomain(_cmd._handle, value);
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "SetDomain Failed with error " + error);
+                    throw ExceptionFactory.CreateException(error);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Gets or Sets pid.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public int Pid
+        {
+            get
+            {
+                int Pid;
+
+                ErrorCode error = VcCmdGetPid(_cmd._handle, out Pid);
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "GetPid Failed with error " + error);
+                    return 0;
+                }
+
+                return Pid;
+            }
+            set
+            {
+                ErrorCode error = VcCmdSetPid(_cmd._handle, value);
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "SetPid Failed with error " + error);
+                    throw ExceptionFactory.CreateException(error);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Gets nlu json data.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string GetNluJson
```

Comment:
Do not use prefix `Get` on get property
```suggestion
        public string NluJson
```

Reply:
sungraejo 5 hours ago  Author Member
We will remove VoiceControlCommandExtended.cs